readthedocs.yml

```diff
@@ -8,6 +8,6 @@ build:
 python:
   version: 3.8
   install:
-    - requirements: requirements_dev.txt
+    - requirements: docs/doc-dependencies.txt
```

Comment:
As per RTD https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html#using-a-configuration-file I suggest changing to docs/requirements.txt

Reply:
Done :smile: